### ISSUE TITLE
no such file to load -- apn_on_rails_tasks 

### ISSUE THREAD

**Issue by michaelkirk** [PRX/apn_on_rails#12]
Following the setup instructions, I've put this in my rakefile:

begin
  require 'apn_on_rails_tasks'
rescue MissingSourceFile => e
  puts e.message
end

But rake is not finding the file:

"no such file to load -- apn_on_rails_tasks "

I have the 0.4.0 version installed in vendor/gems/apn_on_rails-0.4.0/ so I was able to get around this problem by  changing my rakefile to:

begin
  require 'vendor/gems/apn_on_rails-0.4.0/lib/apn_on_rails_tasks'
rescue MissingSourceFile => e
  puts e.message
end

But that's not very convenient. It seems like the gem isn't being included in my path. 

FYI I'm using rails 2.1.

**Comment by rebeccanesson** [comment]
I don't think this require is necessary.  I don't have it in my Rakefile but I have the apn_on_rails rake tasks available.
